app/assets/stylesheets/config/_bootstrap_variables.scss

```diff
@@ -6,7 +6,8 @@
 // General style
```

Comment:
we generally remove unnecessary spaces. Let's use:
`$link-decoration: underline;` instead of
```
$link-decoration:                    underline;
```

Reply:
Fixed!